packages/extension/src/serviceWorker/index.ts

```diff
@@ -134,7 +146,19 @@ chrome.webRequest.onResponseStarted.addListener(
 chrome.webRequest.onBeforeSendHeaders.addListener(
   ({ url, requestHeaders, tabId, frameId, requestId }) => {
     (async () => {
-      const tabUrl = syncCookieStore?.getTabUrl(tabId) ?? '';
+      if (!syncCookieStore) {
```

Comment:
Would you mind adding an explainer comment why `syncCookieStore` not being defined has the conclusion that the SW was in a sleep state?